cni_stop.sh

```diff
@@ -52,6 +52,22 @@ function delete_chache_files {
     rm -rf /var/lib/cni/networks
 }
 
+function unload_ofed_drivers {
+    nic_operator_dir=$WORKSPACE/mellanox-network-operator/deploy
+    operator_namespace=$(grep name $nic_operator_dir/operator-ns.yaml | cut -d: -f2 | tr -d ' ')
+    if [[ -z "$operator_namespace" ]]; then
+        echo "No file named $nic_operator_dir/operator-ns.yaml found. Assuming the operator was not deployed."
+        return 0
+    fi
+    ofed_pod_name=$(kubectl get pods -A -o name | grep ofed | cut -d/ -f2)
+
+    kubectl exec -n "$operator_namespace" -it "$ofed_pod_name" -- /etc/init.d/openibd stop
+    modprobe mlx5_core
```

Comment:
but you loaded them here ...

Reply:
@moshe010, i unload them from inside the ofed container (openibd stop), then reload them from the host machine to load the inbox driver not the ofed one. i do this to prepare the host for the next time the ci will run.

Reply:
so split to 2 functions
1. unload ofed driver
2. reload host drivers